app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 10, load_factor: float = 0.75) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
```

Comment:
Does that make sense to just multiply by 0.75 as this value never changes instead of creating attribute?